bson/src/main/org/bson/internal/UuidHelper.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.bson.internal;
+
+import org.bson.UuidRepresentation;
+import org.bson.BSONException;
+import org.bson.BsonSerializationException;
+import org.bson.BsonBinarySubType;
+
+import java.util.UUID;
+
+/**
+ * Utilities for encoding and decoding UUID into binary.
+ */
+
```

Comment:
nit: floating javadoc - remove the blank line, so the javadoc is directly above the class